bin/main.ml

```diff
@@ -84,6 +104,19 @@ let draw_board_gridlines ctx =
   range 0 width h_spacing (fun col ->
       LTerm_draw.draw_vline ctx 0 col height ~style LTerm_draw.Light)
 
+(* draw letters to letter box given a player's letter [lst] *)
+let rec draw_letters ctx lst =
+  let ctx_size = LTerm_draw.size ctx in
+  match lst with
+  | [] -> ()
+  | h :: t ->
+      if draw_letters ctx t = () then
+        LTerm_draw.draw_string_aligned ctx
+          ((7 - List.length t) * (ctx_size.rows / 8))
+          H_align_center
+          (Zed_string.of_utf8 (String.make 1 h))
```

Comment:
it could be simpler to use `LTerm_draw.draw_char`; then you would not have to use `String.make 1`. this doesn't allow you to center the letters so easily, though. what do you think?

Reply:
I will try it out and see what the results are